ClientGenerator/src/googleapis/codegen/languages/csharp/default/templates/___package_name___/___package_name___.csproj.tmpl

```diff
@@ -27,7 +27,7 @@
 {% endnoblank %}
       Supported Platforms:
       - .NET Framework 4.5+
-      - NetStandard1.3, providing .NET Core support.
+      - NetStandard1.3 and NetStandard2.0; providing .NET Core support.
```

Comment:
We should reword this to .NET Standard 1.3 and .NET Standard 2.0. netstandard1.3 and netstandard2.0 are fine as TPMs, but by the time we're capitalizing, we're into product name territory.

Reply:
done